src/Plugin/Field/FieldFormatter/AdvancedEntityReferenceFormatter.php

```diff
@@ -1,32 +1,34 @@
 <?php
 
-namespace Drupal\entity_reference_display_switcher\Plugin\Field\FieldFormatter;
+namespace Drupal\advanced_entity_reference_formatter\Plugin\Field\FieldFormatter;
 
 use Drupal\Component\Utility\NestedArray;
 use Drupal\Core\Field\FieldItemListInterface;
 use Drupal\Core\Field\Plugin\Field\FieldFormatter\EntityReferenceEntityFormatter;
 use Drupal\Core\Form\FormStateInterface;
 
 /**
- * Plugin implementation of the 'ParagraphsViewModeSwitcherFormatter' formatter.
+ * Plugin implementation of the 'AdvancedEntityReferenceFormatter' formatter.
  *
  * @FieldFormatter(
- *   id = "entity_reference_display_switcher",
- *   label = @Translation("Entity reference display switcher"),
+ *   id = "advanced_entity_reference_formatter",
+ *   label = @Translation("Advanced Entity Reference Formatter"),
  *   field_types = {
  *     "entity_reference",
  *     "entity_reference_revisions"
  *   }
  * )
  */
-class EntityReferenceDisplaySwitcherFormatter extends EntityReferenceEntityFormatter {
+class AdvancedEntityReferenceFormatter extends EntityReferenceEntityFormatter {
 
   /**
    * {@inheritdoc}
    */
   public static function defaultSettings() {
     return [
       'view_mode' => 'default',
+      'override' => 'none',
+      'bundle_view_mode' => [],
       'view_mode_delta' => [],
       'link' => FALSE,
```

Comment:
`view_mode` and `link` settings come from parent so it might be better to remove them here.

Reply:
fixed